app/helpers/plants_helper.rb

```diff
@@ -3,7 +3,11 @@
 module PlantsHelper
   def plant_img(plant)
     if plant.img.present?
-      image_tag plant.img
+      image_tag plant.img.variant(
+        saver: { quality: 50},
+        convert: "jpg",
+        resize_to_fill: [400, 400]
+      )
```

Comment:
This is the crux of the PR, what is required is the uniformity of image sizes regardless of what is uploaded.